### ISSUE TITLE
Too many leftovers

### ISSUE THREAD

**Issue by cdedwards** [Duet3D/DWC-CNC#1]
I'm really hoping this is continued as the idea is great. I'm currently building a router which will be using a Duet2. I'm also building a CO2 laser which will use a Duet3 with SBC. With either of these two systems, there are features of the DWC which are not needed or required. Filament, Layer chart, Extrusion, and Fan control. Collected Data could be modified for CNC operations.. Babystepping might be useful.  Tool, Bed and Chamber Temps are not needed or required but are hidden on another screen, so not as important as ripping out the other's. I'm not familiar with vue at all to even begin on helping here, but I may end up learning.

**Comment by Sindarius** [comment]
I agree there is still plenty to do here. The initial go was focused on cleaning up the dashboard. However, like all things I have been focused on a couple other tasks to put more time on this one. There are several todo items that I would like to get back to on this. Several requests are on the forums.
